app/gui/language/span-tree/src/generate.rs

```diff
@@ -153,27 +153,41 @@ struct ApplicationBase<'a> {
     function_name: Option<&'a str>,
     /// True when Ast uses method notation to pass this as an invocation target.
     has_target:    bool,
+    /// The ast id of the call expression.
+    call_id:       Option<Id>,
+    /// The id of the call target operand. The target can present with not assigned ast id.
```

Comment:
Is the sentence "The target can present with not assigned ast id." a corrct english one? I feel its not.